frontend/src/components/Category/CategoryTag.jsx

```diff
@@ -1,10 +1,14 @@
 import './CategoryTag.css';
 
 const CategoryTag = ({ category }) => {
-  const cssSuffix = category.toLowerCase().replace(/\s/g, '');
+  const { long_name, short_name, color } = category;
   return (
-    <span className={`category-tag category-tag-${cssSuffix}`}>
-      {category}
+    <span
+      className={`category-tag`}
+      style={{ background: color }}
+      title={long_name} // TODO make a better tooltip
```

Comment:
Wow. I did not know you could tooltip like this.

Reply:
it's a very basic tooltip though.
Mui probably has better ones we could use